src/Liquidator_NEW.sol

```diff
@@ -195,5 +196,39 @@ contract Liquidator_NEW is Owned {
         ILendingPool_NEW(creditor).startLiquidation(account, debt);
 
         // Fill the auction struct
+        auctionInformation[account].startPrice = _calculateStartPrice(debt);
+        auctionInformation[account].startTime = uint32(block.timestamp);
+        auctionInformation[account].assetShares = _getAssetDistribution(riskValues);
+
+        // Emit event
+        emit AuctionStarted(account, creditor, assetAddresses[0], uint128(debt));
+    }
+
+    function _calculateStartPrice(uint256 debt) internal view returns (uint256 startPrice) {
+        startPrice = debt * startPriceMultiplier / 100;
+    }
+
+    function _getAssetDistribution(RiskModule.AssetValueAndRiskVariables[] memory riskValues_)
+        internal
+        pure
+        returns (uint16[] memory assetDistributions)
+    {
+        uint256 length = riskValues_.length;
+        uint256 totalValue;
+        for (uint256 i; i < length;) {
+            totalValue += riskValues_[i].valueInBaseCurrency;
```

Comment:
Can be in unchecked, we already know from checkAndStartLiquidation() that this sum does not overflow.